app/styles/tasks.css

```diff
@@ -1,11 +1,13 @@
 :root {
-  --slider-colour: #424CC1;
-  --slider-background: #CFD3FF;
-  --thumb-background: #A1D0FF;
+  --slider-colour: #424cc1;
+  --slider-background: #cfd3ff;
+  --thumb-background: #a1d0ff;
   --opaque-black: #000000b3;
   --white: #ffffff;
   --black: #000000;
   --green: #008000;
+  --light-black : rgba(0, 0, 0, 0.3);
+  --gray: #f1f1f1;
```

Comment:
Can we please move this to `variables.css` and then use it here, also do check if there is any unused variable, we can delete that.